main.py

```diff
@@ -1,7 +1,30 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+    for genre in ["Western", "Action", "Dramma"]:
+        Genre.objects.create(name=genre)
```

Comment:
add a blank line between two logical parts of the code, you can simply insert a blank line to visually separate the sections.